src/main/controller/ExerciseController.ts

```diff
@@ -0,0 +1,22 @@
+import { Request, Response } from 'express';
+import db from '../models';
+
+export class ExerciseController {
+  getAllExercises = async (_req: Request, res: Response) => {
```

Comment:
Could you add the return types for all the controller methods? I think you missed a few!